src/client/utils/validations.js

```diff
@@ -318,8 +319,10 @@ function isValidChildren(data) {
     }
   }
 
-  return isNotBlank(data.hasChildrenToReport.value) &&
-      allChildrenValid;
+  return data.understandsFinancialDisclosure || (
```

Comment:
This check for the financialDisclosure doesn't belong in this function, this function has to do with the child information fields not the child income/expense fields.